### PR TITLE
Rework the docker images

### PR DESCRIPTION
**Pull request by john-floren-gravwell**
Per #113 and #110, our build process did a lot of excess work. These changes simplify the Docker build steps to mostly package up our existing Docker images with custom configs where needed. It also reworks the training document to deal with the tweaks.

### CODE REVIEW

File: dockerfiles/createslim.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ if [ ! -f "$LICENSE" ]; then
 fi
 
 docker rmi gravwell:slim #remove existing slim image
-docker pull gravwell/gravwell:latest # grabs latest gw image from dockerhub
-docker create --name slim gravwell/gravwell:latest #create temp container from latest image
+docker pull gravwell/gravwell:${VER} # grabs latest gw image from dockerhub
```
Comment (ashnwade) [inline dockerfiles/createslim.sh:8]: Can we continue to use latest?
If not, we should probably change the comment here that says "latest"

File: dockerfiles/build_all.sh [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
+set -e
 LOGFILE=/tmp/build.log
 OUTDIR=../dockerimages/
-VER=${VERSION:-5.3.0}
+VER=${VERSION:-5.4.10}
```
Comment (ashnwade) [inline dockerfiles/build_all.sh:5]: Can we use latest?

**Review by ashnwade — COMMENTED** [review]
Can we always use the latest version?
We don't have a large tail on dockerhub currently: `5.4.9`, `5.4.10`, `5.4.11`, `latest`
I feel like this could become broken/outdated pretty quickly if we tie it closely to a particular version. We always have a latest version though.

I think we also need to update the README in the root directory.

**Comment by john-floren-gravwell** [comment]
> We still need to update the README in the root directory. If we are listing a version there, it should also use latest, but more importantly, I thought the point of this change was to use docker packages without having to build, which is what the README is still telling you to do.
> 
> If I try to follow the README for what I am supposed to do, then I get an error "Must set GOPATH". If you are still going to have build instructions in the README, then maybe we need more details to (1) indicate that this is optional and that you can use public containers instead (?) and/or (2) provide more details about setting GOPATH as a prerequisite.

We build Docker images for a few reasons:

* So we can bundle in a license, to avoid tedious uploading for every lab
* So we can pre-configure various components
* So trainees in a corporate environment don't experience throttling while trying to fetch from Dockerhub.

I will add a note about GOPATH.

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**